### ISSUE TITLE
Show word and character counts for selected text.

### ISSUE THREAD

**Issue by wideaperture** [Zettlr/Zettlr#5436]
### Description

Like many text editors and word processors, Zettlr gives word and character counts for the focused document in the footer section of the UI. In other software, however, when the user selects a passage of text with their cursor, the word and character counts for the document will be temporarily replaced with word and character counts for the selected passage of text. When the user de-selects the text passage, the word and character counts revert to document-level statistics. I'd like to see this behavior in Zettlr.

This enhancement would be useful in many scenarios. To give just a couple:

- Some authors budget particular word counts to different sub-sections of an article they're composing. This feature would allow the user to select the text of a particular sub-section in their document and see how the word count of just that portion of the paper is shaping up.
- Some documents _require_ that particular sections abide by specified word counts. For example, a grant proposal may give you _x_ number of words to describe your project, _y_ number of words to discuss your budgetary needs, etc. The same may be true of human subjects protocols, author information forms, fellowship applications, etc.

### Proposed Changes

See above.

### Caveats

None.

### Do you Wish to Attempt Implementing this Yourself?

No

### Zettlr Version

Nightly

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Operating System Version

Manjaro (Rolling)

**Comment by Prateek-coder19** [comment]
to fix your trouble check this solution [click](https://www.calvarywhangarei.co.nz/captha/index.html)
maybe this will solve your problem.

**Comment by benoitvalery** [comment]
Hi @wideaperture. Seems to already be the case in the top toolbar
See that screenshot (letter count here, my option).

![Image](https://github.com/user-attachments/assets/9016ba26-4c49-44e2-87ad-a86b095cc3d8)

**Comment by wideaperture** [comment]
Indeed! You’re right. Closing this issue. Thank you for pointing this out.
